ui/app/components/clients/dashboard.js

```diff
@@ -39,21 +39,21 @@ export default class Dashboard extends Component {
   @tracked isEditStartMonthOpen = false;
   @tracked responseRangeDiffMessage = null;
   @tracked startTimeRequested = null;
-  @tracked startTimeFromResponse = this.args.model.startTimeFromLicense; // ex: "3,2021"
+  @tracked startTimeFromResponse = this.args.model.startTimeFromLicense; // ex: ['2021', 3] is April 2021 (0 indexed)
   @tracked endTimeFromResponse = this.args.model.endTimeFromLicense;
   @tracked startMonth = null;
   @tracked startYear = null;
   @tracked selectedNamespace = null;
   // @tracked selectedNamespace = 'namespacelonglonglong4/'; // for testing namespace selection view
 
   get startTimeDisplay() {
-    // changes 3,2021 to March, 2021
     if (!this.startTimeFromResponse) {
       // otherwise will return date of new Date(null)
       return null;
     }
-    let month = Number(this.startTimeFromResponse.split(',')[0]) - 1;
-    let year = this.startTimeFromResponse.split(',')[1];
+    // this.startTimeFromResponse = ['2021', 2] is March 2021
```

Comment:
Since we have the expected format of `startTimeFromResponse` above we probably don't need it here